### PR TITLE
feat(Autocomplete): add algolia-insights example

### PR DESCRIPTION
**Pull request by tkrugg**
Moving Autocomplete example out of search-insights repo.
https://github.com/algolia/search-insights.js/tree/develop/examples/autocomplete

### CODE REVIEW

File: Autocomplete.js/algolia-insights/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    </svg>
+</div>
+
+<script src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearch.min.js"></script>
+<script src="https://cdn.jsdelivr.net/autocomplete.js/0/autocomplete.min.js"></script>
+<script src="src/app.js"></script>
+
+<script>
+    !function(e,a,t,n,s,i,c){e.AlgoliaAnalyticsObject=s,e.aa=e.aa||function(){(e.aa.queue=e.aa.queue||[]).push(arguments)},i=a.createElement(t),c=a.getElementsByTagName(t)[0],i.async=1,i.src="https://cdn.jsdelivr.net/npm/search-insights@1.0.1",c.parentNode.insertBefore(i,c)}(window,document,"script",0,"aa");
+    aa('init', {
```
Comment (Haroenv) [inline Autocomplete.js/algolia-insights/index.html]: shouldn't this be loaded before the other scripts?

Comment (tkrugg) [reply Autocomplete.js/algolia-insights/index.html]: hum, you're right. Move it in the header.

File: Autocomplete.js/algolia-insights/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            autocomplete="off"
+    />
+    <svg class="aa-input-icon" viewBox="654 -372 1664 1664">
+        <path
+                d="M1806,332c0-123.3-43.8-228.8-131.5-316.5C1586.8-72.2,1481.3-116,1358-116s-228.8,43.8-316.5,131.5  C953.8,103.2,910,208.7,910,332s43.8,228.8,131.5,316.5C1129.2,736.2,1234.7,780,1358,780s228.8-43.8,316.5-131.5  C1762.2,560.8,1806,455.3,1806,332z M2318,1164c0,34.7-12.7,64.7-38,90s-55.3,38-90,38c-36,0-66-12.7-90-38l-343-342  c-119.3,82.7-252.3,124-399,124c-95.3,0-186.5-18.5-273.5-55.5s-162-87-225-150s-113-138-150-225S654,427.3,654,332  s18.5-186.5,55.5-273.5s87-162,150-225s138-113,225-150S1262.7-372,1358-372s186.5,18.5,273.5,55.5s162,87,225,150s113,138,150,225  S2062,236.7,2062,332c0,146.7-41.3,279.7-124,399l343,343C2305.7,1098.7,2318,1128.7,2318,1164z"
+        />
+    </svg>
+</div>
+
+<script src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearch.min.js"></script>
```
Comment (francoischalifour) [inline Autocomplete.js/algolia-insights/index.html]: We should use pinned versions.

File: Autocomplete.js/algolia-insights/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<script src="https://cdn.jsdelivr.net/autocomplete.js/0/autocomplete.min.js"></script>
```
Comment (francoischalifour) [inline Autocomplete.js/algolia-insights/index.html]: We should use pinned versions.

File: Autocomplete.js/algolia-insights/package.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "parcel-sandbox",
```
Comment (francoischalifour) [inline Autocomplete.js/algolia-insights/package.json]: Might want to update this.

File: Autocomplete.js/algolia-insights/package.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  "version": "1.0.0",
+  "description": "Simple Parcel Sandbox",
```
Comment (francoischalifour) [inline Autocomplete.js/algolia-insights/package.json]: Might want to update this.

File: Autocomplete.js/algolia-insights/package.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/core": "7.2.0",
+    "parcel-bundler": "^1.6.1",
```
Comment (francoischalifour) [inline Autocomplete.js/algolia-insights/package.json]: We pin those usually.

File: Autocomplete.js/algolia-insights/package.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    "prettier": "^1.16.4"
```
Comment (francoischalifour) [inline Autocomplete.js/algolia-insights/package.json]: We pin those usually.

**Pull request closed without merging**